src/docx/index.js

```diff
@@ -72,44 +94,68 @@ module.exports = {
   generateDocument
 };
 
-if (require.main === module) {
-  generateDocument("INDYWIDUALNE", {
-    applicant: {
-      address: "Wielka 4/6, 55-789 Studnia",
-      name: "Tomasz Rzeźniczak i Adelajda Kieł"
-    },
-    child: {
-      address: "55-789 Studnia, Wielka 4/6",
-      birthDate: "12.12.1912",
-      birthPlace: "Pachy Wielkie",
-      name: "Wiktor Eridfasdfz2",
-      pesel: "12121244441"
-    },
-    city: "Grodzisk Wielkopolski",
-    date: "15.07.2019",
-    no: "42",
-    parents: {
-      address: "65-898 Ugody, Zielona 29b/5",
-      name: "Tomasz Rzeźniczak i Adelajda Kieł"
-    },
+var specificData = {
+  INDYWIDUALNE: {
     period: "styczeń 2018 - listopad 2019",
-    reason: "ZNACZNIE_UTRUDNIAJACY",
-    school:
-      "Szkoła Podstawowa nr 4 w Grodzisku Wlkp., szkoła podstawowa, 62-065 Grodzisk Wlkp., ul. Środkowa 56, murarz, 5a",
-    team: {
-      address: "ul. Zbąszyńska 11",
-      members: [
-        { name: "Antoni Stąsz-Lebieź", function: "przewodniczący zespołu" },
-        { name: "mgr Leonia Witek-Konuś", function: "psycholog, tyflopedagog" },
-        {
-          name: "mgr Eleonora Roseveelt",
-          function: "socjoterapeuta, tyflopedagog"
-        }
-      ],
-      name:
-        "Zespół Orzekający przy Poradni Psychologiczno-Pedagogicznej w Grodzisku Wlkp."
-    }
-  })
+    reason: "ZNACZNIE_UTRUDNIAJACY"
+  },
+  SPECJALNE: {
+    period: "pierwszy etap edukacyjny",
+    reason: "SPRZEZONA",
+    multipleDisability: [
+      "niesłyszące",
+      "niepełnosprawne intelektualnie w stopniu znacznym"
+    ]
+  }
+};
+
+var commonData = {
+  applicant: {
+    address: "Wielka 4/6, 55-789 Studnia",
+    name: "Tomasz Rzeźniczak i Adelajda Kieł"
+  },
+  child: {
+    address: "55-789 Studnia, Wielka 4/6",
+    birthDate: "12.12.1912",
+    birthPlace: "Pachy Wielkie",
+    name: "Wiktor Eridfasdfz2",
+    pesel: "12121244441"
+  },
+  city: "Grodzisk Wielkopolski",
+  date: "15.07.2019",
+  no: "42",
+  parents: {
+    address: "65-898 Ugody, Zielona 29b/5",
+    name: "Tomasz Rzeźniczak i Adelajda Kieł"
+  },
+  school:
+    "Szkoła Podstawowa nr 4 w Grodzisku Wlkp., szkoła podstawowa, 62-065 Grodzisk Wlkp., ul. Środkowa 56, murarz, 5a",
+  supportCenter: {
+    address: "ul. Zbąszyńska 11",
+    members: [
+      {
+        name: "Antoni Stąsz-Lebieź",
+        function: "przewodniczący zespołu"
+      },
+      {
+        name: "mgr Leonia Witek-Konuś",
+        function: "psycholog, tyflopedagog"
+      },
+      {
+        name: "mgr Eleonora Roseveelt",
+        function: "socjoterapeuta, tyflopedagog"
+      }
+    ],
+    name:
+      "Zespół Orzekający przy Poradni Psychologiczno-Pedagogicznej w Grodzisku Wlkp.",
+    post: "62-065 Grodzisk Wlkp."
+  },
+  kurator: "w Poznaniu, ul Kościuszki 38, 64-400 Poznań"
+};
+
+if (require.main === module) {
+  Object.assign(commonData, specificData[process.argv[2]]);
```

Comment:
I would also instead of repeating `process.argv[2]` set it to constant `const documentType = process.argv[2];`